articles/clients/grant-types/client-grant-types.md

```diff
@@ -0,0 +1,124 @@
+* `http://auth0.com/oauth/legacy/grant-type/delegation/refresh_token`
+* `http://auth0.com/oauth/legacy/grant-type/delegation/id_token`
+* `http://auth0.com/oauth/legacy/grant-type/access_token`
+
+## Edit the `grant_type` Property
+
+To edit the `grant_type` property for your Auth0 Client, you'll need to make the appropriate call to the [Update a Client endpoint](/api/management/v2#!/Clients/patch_clients_by_id) of the [Management API](/api/management/v2).
+
+:::panel-warning Legacy Grant Types
+New Auth0 customers **cannot** add *any* of the legacy grant types to their Clients. Only customers as of TBD can add legacy grant types to their existing Clients.
+:::
+
+Auth0 requires the `grant_type` property, so using a Client *without* this property populated results in errors.
+
+### Existing Clients
+
+For existing Clients, Auth0 will populate the `grant_type` property with the correct value so that this change doesn't affect functionality.
+
+### New Clients
+
+At creation, new Clients (regardless of whether the owning tenant exists or is new), will be assigned the appropriate default grant type depending on the following two Client flags:
+
+* `oidc_conformant`: This flag is `true` if you've enabled the [**OIDC Conformant** slider](/api-auth/tutorials/adoption/oidc-conformant) in the *Advanced Settings* section of your Client Settings page.
+* `is_first_party`: By default, all new Clients will have this flag set to `true`.
+
+If `is_first_party` is `true`, the Client can use the following grant types (regardless of whether `oidc_conformant` is `true` or `false`):
+
+* `implicit`
+* `authorization_code`
+* `client_credentials`
+* `password`
+* `refresh_token`
+* `http://auth0.com/oauth/grant-type/password-realm`
+* `http://auth0.com/oauth/grant-type/mfa-oob`
+* `http://auth0.com/oauth/grant-type/mfa-otp`
+* `http://auth0.com/oauth/grant-type/mfa-recovery-code`
+
+If `is_first_party` is `false` **and** the `oidc_conformant` flag is `true`, the Client can use the following grant types:
+
+* `implicit`
+* `authorization_code`
+* `client_credentials`
+* `refresh_token`
+* `http://auth0.com/oauth/grant-type/mfa-oob`
+* `http://auth0.com/oauth/grant-type/mfa-otp`
+* `http://auth0.com/oauth/grant-type/mfa-recovery-code`
+
+For third-party Clients, Auth0 limits used of the Password Exchange flows by default. To use this flow, you must explicitly enable it by making the appropriate `PATCH` call to the [Update a Client endpoint](/api/management/v2#!/Clients/patch_clients_by_id) of the [Management API](/api/management/v2).
+
+## Secure Alternatives to the Legacy Grant Types
+
+<table class="table">
+  <tr>
+    <th>If you want to use this legacy grant type...</th>
+    <th>Consider using...</th>
+  </tr>
+  <tr>
+    <td>`http://auth0.com/oauth/legacy/grant-type/ro`</td>
+    <td>`http://auth0.com/oauth/grant-type/password-realm`</td>
+  </tr>
+  <tr>
+    <td>`http://auth0.com/oauth/legacy/grant-type/ro/jwt-bearer`</td>
+    <td></td>
```

Comment:
Sorry, should have been clearer...

I meant that for cases such as `http://auth0.com/oauth/legacy/grant-type/ro/jwt-bearer` where there is no current replacement secure feature available, we can say `Contact Support` (right now it's just an empty cell). Or, if you think it's better, we could say more like "If current customer as of (date), patch using APIv2 to use this. If new customer after (date), contact support."